lib/context.js

```diff
@@ -387,13 +403,65 @@ class ResponseContext {
         await regenerateSessionStoreId(req, config);
       }
       resumeSilentLogin(req, res);
+
+      if (
+        req.oidc.isAuthenticated() &&
+        config.backchannelLogout &&
+        config.backchannelLogout.onLogin !== false
+      ) {
+        await (config.backchannelLogout.onLogin || onLogin)(req, config);
+      }
     } catch (err) {
       if (!req.openidState || !req.openidState.attemptingSilentLogin) {
         return next(err);
       }
     }
     res.redirect(req.openidState.returnTo || config.baseURL);
   }
+
+  async backchannelLogout() {
+    let { config, req, res } = weakRef(this);
+    res.setHeader('cache-control', 'no-store');
+    const logoutToken = req.body.logout_token;
+    if (!logoutToken) {
+      res.status(400).json({
+        error: 'invalid_request',
+        error_description: 'Missing logout_token',
+      });
+      return;
+    }
+    const onToken =
+      (config.backchannelLogout && config.backchannelLogout.onLogoutToken) ||
```

Comment:
It looks like people can set `onLogin` to false, and we wont even run our default onLogin.
Is there a reason we dont have this for onLogoutToken? Or in general, any reason why it's not behaving the same one way or the other?

Reply:
Yes - you can't implement back-channel logout without doing something with the logout token (onLogoutToken), but you can implement back-channel logout without doing anything when the user logs in (onLogin)

See these 2 custom implementations as an example:
https://github.com/auth0/express-openid-connect/blob/back-channel-logout/examples/backchannel-logout-custom-query-store.js#L46-L48
https://github.com/auth0/express-openid-connect/blob/back-channel-logout/examples/backchannel-logout-custom-genid.js#L30-L34